aries-site/src/pages/components/datafilter.mdx

```diff
@@ -0,0 +1,20 @@
+import { DataFilterExample } from '../../examples/components/';
+import { Example } from '../../layouts';
+
+Based on the type of data for that property, DataFilter determines a suitable "smart default" for what kind of input to present. The "smart default" is determined based on the following criteria:
+
+| Type       | Number of options    | Default filter type        |
+| ---------- | -------------------- | -------------------------- |
+| **string** | Less than 5 options  | CheckBoxGroup              |
+| **string** | 5-9 options          | SelectMultiple             |
+| **string** | More than 10 options | SelectMultiple with search |
+| **number** | --                   | RangeSelector              |
+
+<Example
```

Comment:
To follow general page convention, can we move this example above the rest of the guidance?